### DISCUSSION TITLE
Bar graph doesn't show ordinal values on primary axis

### DISCUSSION THREAD

**Discussion by mrlinux1** [TanStack/react-charts#104] [General] [answered]
Hello, 
I try to create a stacked bar graph like in the example.
I remapped the data so that it has the same layout as in the example bar chart:
![data_comparison](https://user-images.githubusercontent.com/9421663/88361763-660c2e80-cd7a-11ea-801c-a64344cd53d1.png)
But my graph doesn't show the names on the left side of the graph:
![graph_comparison](https://user-images.githubusercontent.com/9421663/88361777-73c1b400-cd7a-11ea-8a53-910243a2f159.png)

I use the same code snippets for my graph like in the example bar chart (data, series, axes).

Can anyone help me on that?
Kind regards,
Thomas

**Comment by mrlinux1** [comment]
Now I moved the example graph to my component and use the same series and axes memo:

 ```
     const axes = React.useMemo(
        () => [
          { primary: true, type: 'ordinal', position: 'left', primaryAxisShow: true },
          { type: 'linear', position: 'bottom', stacked: true, secondaryAxisShow: true},
        ],
        []
      )
      const series = React.useCallback(
        () => ({
          
          type: 'bar'
        }),
        []
      )
```
But I get the same result.
But the data looks the same:
![react](https://user-images.githubusercontent.com/9421663/88622025-21e29c00-d0a2-11ea-8aef-c16788fe6213.png)

I'm really frustrated as I need that until next weekend.

Kind regards,

Thomas

**Comment by evenstephenr** [comment]
@mrlinux1 I took a look at the react-charts internal API and it doesn't look like ordinal groups let you customize the axis label ☹️ 

To clarify for my own understanding, are you looking to replace the 'ordinal group' label with a categorical group label - in your case, country name?

**Reply by mrlinux1** [reply]
@evenstephenr Yes, that what I try to achieve. I tried to figure out how the example sets the y-axis but I can't get it to work with my data.

**Comment by tannerlinsley** [comment, marked as answer]
Due to a massive overhaul and rewrite of React Charts to TypeScript and a massively upgraded public API, this discussion is being marked as answered for triage. The latest version can be tested via the `react-charts@beta` tag. Documentation is in the process of being upgraded, but thankfully, everything is fully typed now, so you can explore the API that way for now.

If, *after testing or playing with the v3 beta*, you believe this discussion is still relevant or unanswered, feel free to mark it as such and continue the discussion.
